src/docs/versum/Materia.mdx

```diff
@@ -7,6 +7,7 @@ description: introduction to Materia
 
 Materia (`MTRIA`) is Versum's utility token.
 As a participant of the Tezos community, you are eligible to receive Materia. The amount is calculated based on your historic activity.
+This historic activity is processed daily and the reason you might not have materia to claim is because we might have included your transactions.
```

Comment:
```suggestion
Historic activity is processed daily. There will be times when you won't have new Materia to claim because those transactions were already included as part of a previous calculation and allocation to you, or because our processing hasn't got to your transactions yet.
```